config/initializers/maintenance_mode.rb

```diff
@@ -0,0 +1,2 @@
+Rails.application.config.x.maintenance_mode = ENV["MAINTENANCE_MODE"].present?
```

Comment:
I've just looked up the docs for custom configuration and it looks like we only need to use `config.x` if we plan to nest configuration. In which case can we use the simpler `.config` instead?